docs/reference/technology/firmware_update.md

```diff
@@ -1,96 +1,81 @@
 
-- Mbed OS now includes the bootloader that manages update verification and installation.
-- Mbed CLI and the Mbed Online Compiler now support update actions using the Update Service API and manifest tool. We have a quick start guide [for updating through the Online Compiler](https://cloud.mbed.com/guides/pelion-firmware-update), and a review [of the Mbed CLI flow and commands](../tools/cli-update.html).
+Mbed CLI and the Mbed Online Compiler now support update actions using the Update Service API and manifest tool. We have a quick start guide [for updating through the Online Compiler](https://cloud.mbed.com/guides/pelion-firmware-update), and a review [of the Mbed CLI flow and commands](../tools/cli-update.html).
 
 ## Updatable binaries
 
 The active firmware, made up of Mbed OS and an application, can be updated only if the binary installed on the device has:
 
-1. A bootloader, as [reviewed below](#the-mbed-os-bootloader). The bootloader can load a new version of the firmware. If for some reason, the update fails, the bootloader falls back to the firmware version last known to work. As a security feature, the default behavior of the bootloader is to refuse to roll back to old firmware versions once an update succeeds.
-1. Mbed OS with the Device Management Client (which includes the Update client). The clients allow your device to communicate with the Device Management Update service, receive update manifests and firmware, and verify the binary's validity. They are [reviewed in details in our Pelion Device Management documentation](https://cloud.mbed.com/docs/latest/updating-firmware/index.html).
+1. A boot loader, as [reviewed below](#the-mbed-os-bootloader). The boot loader can load a new version of the firmware. As a security feature, the default behavior of the boot loader is to refuse to roll back to old firmware versions once an update succeeds.
+1. Mbed OS with Device Management Client (which includes the Update client). The clients allow your device to communicate with the Device Management Update service, receive update manifests and firmware, and verify the binary's validity. They are [reviewed in details in our Pelion Device Management documentation](https://www.pelion.com/docs/device-management/current/updating-firmware/index.html).
 
-  <span class="notes">**Note:** Because some embedded devices don't require remote update capabilities, Mbed OS does not include the Device Management Client by default. You must explicitly import the client to your application.</span>
+<span class="notes">**Note:** Because some embedded devices don't require remote update capabilities, Mbed OS does not include the Device Management Client by default. You must explicitly import the client to your application.</span>
 
 1. Permission to access your Device Management account, as well as keys and certificates to verify the firmware's source and validity.
 
-## The Mbed OS bootloader
+## The Mbed OS boot loader
```

Comment:
The phrase `The Mbed OS boot loader` is a bit vague. I would be specific on the separate concepts:

- Mbed OS bootloader framework
- Pelion Bootloader - [mbed-bootloader](https://github.com/ARMmbed/mbed-bootloader)
- Mbed OS Bootloader Example - [mbed-os-example-bootloader](https://github.com/ARMmbed/mbed-os-example-bootloader)